src/NServiceBus.AcceptanceTests/UnitOfWork/TransactionScope/When_used_with_default_transaction_mode.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace NServiceBus.AcceptanceTests.UnitOfWork
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using Configuration.AdvanceExtensibility;
+    using EndpointTemplates;
+    using FakeTransport;
+    using NUnit.Framework;
+
+    public class When_used_with_default_transaction_mode : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_work()
+        {
+            var context = await Scenario.Define<Context>()
+                 .WithEndpoint<ScopeEndpoint>(b => b.CustomConfig(c =>
+                 {
+                     c.GetSettings().Set("FakeTransport.SupportedTransactionMode", TransportTransactionMode.ReceiveOnly);
```

Comment:
Sorry, yes I meant `When_used_with_transport_scopes`.

I tried a unit test first but testing the feature setup methods are tricky (so much stuff to setup, I know that @danielmarbach have propose that we invest some time to create a feature test harness)